src/category_theory/epi_mono.lean

```diff
@@ -2,15 +2,17 @@
 Copyright (c) 2019 Reid Barton. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Reid Barton, Scott Morrison
+-/
+import category_theory.adjunction.basic
+import category_theory.opposites
+import category_theory.groupoid
 
-Facts about epimorphisms and monomorphisms.
+/-!
+# Facts about epimorphisms and monomorphisms.
```

Comment:
What are epimorphisms and monomorphisms?

Reply:
They are defined in an earlier file, and the doc-strings already contain the definitions.

Reply:
Are you talking about `category_theory.category.default`? Because there's no explanation of what epimorphisms and monomorphisms are in the module docstring. The present file is about epi/monomorphisms and it's not exaggerated to expect a definition here, especially because the Lean definition itself isn't there!
Furthermore, but that's an orthogonal problem. `.default` files should be importing the other files in their folder and do nothing else. I wasn't expecting any content in there. This sounds like `category_theory.category.default` should be `category_theory.basic`.

Reply:
Sure, feel free to rename.

The definitions are in the doc-string on the actual declarations. Not definition and lemma needs to be described in the module doc.